GiEnJul/ClientApp/src/components/registerGiver/Macro.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from "react";
+import { CssBaseline, Typography, Container } from "@material-ui/core";
+import Confirmation from "./Confirmation";
+import ContactInfo from "./ContactInfo";
+import Location from "./Location";
+import SummaryRegistration from "./Summary";
+import IFormData from "./IFormData";
+import FamilySize from "./FamilySize";
+import getLocations from "../../common/constants/Locations";
+
+const initFormDataState: IFormData = {
+  location: "",
+  fullname: "",
+  email: "",
+  phoneNumber: "",
+  maxReceivers: "",
+};
+
+const initState: { step: number; confirmationOK?: boolean } = {
+  step: 1,
+  confirmationOK: undefined,
+};
+
+const SignUp = () => {
```

Comment:
I like the renaming from SignUp to Macro, but should it be more descriptive? SignUpMacro | RegistrationMacro, or something along those lines?
line 24 has to be changed to match the name of the file. Same as the export at line 151

Reply:
Yeah I thought about this, but I feel that Macro is enough, because everything in the folder is for the registration/signup.

Reply:
The component should have another name though, as it's used elsewhere as well. I'll change it to RegistrationMacro.

Reply:
Yea, sounds like a good plan

Reply:
c76abe1 only removed unecessary props. Did you forget a file?

Reply:
Whoops I did 😱 Will fix!